index.html

```diff
@@ -2,85 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calendar</title>
+    <script defer src="todo.js"></script>
     <link rel="stylesheet" href="style.css" />
-    <title>To Do List</title>
   </head>
   <body>
-    <div class="calender">
-      <h1>Planner</h1>
-      <h2>Calender</h2>
+    <div id="container">
+      <div id="header">
+        <div id="monthDisplay"></div>
+        <div>
+          <button id="backButton">&#10094;</button>
+          <button id="nextButton">&#10095;</button>
+        </div>
+      </div>
 
-      <div class="month">
-        <ul>
-          <li class="prev">&#10094;</li>
-          <li class="next">&#10095;</li>
-          <li>May<br /><span style="font-size: 22px">2022</span></li>
-        </ul>
+      <div id="weekdays">
+        <div>Sunday</div>
+        <div>Monday</div>
+        <div>Tuesday</div>
+        <div>Wednesday</div>
+        <div>Thursday</div>
+        <div>Friday</div>
+        <div>Saturday</div>
       </div>
 
-      <ul class="weekdays">
-        <li>Su</li>
-        <li>Mo</li>
-        <li>Tu</li>
-        <li>We</li>
-        <li>Th</li>
-        <li>Fr</li>
-        <li>Sa</li>
-      </ul>
+      <div id="calendar"></div>
+    </div>
+
+    <div id="newEventModal">
+      <h2>To Do</h2>
 
-      <ul class="days">
-        <li>1</li>
-        <li>2</li>
-        <li>3</li>
-        <li>4</li>
-        <li>5</li>
-        <li>6</li>
-        <li>7</li>
-        <li>8</li>
-        <li>9</li>
-        <li>10</li>
-        <li><span class="active">11</span></li>
-        <li>12</li>
-        <li>13</li>
-        <li>14</li>
-        <li>15</li>
-        <li>16</li>
-        <li>17</li>
-        <li>18</li>
-        <li><span class="bday">&#127874;</span></li>
-        <li>20</li>
-        <li>21</li>
-        <li>22</li>
-        <li>23</li>
-        <li>24</li>
-        <li>25</li>
-        <li>26</li>
-        <li>27</li>
-        <li>28</li>
-        <li>29</li>
-        <li>30</li>
-      </ul>
+      <input id="eventTitleInput" placeholder="Add Item...." />
+
+      <button id="saveButton">Save</button>
+      <button id="cancelButton">Cancel</button>
     </div>
-    <div class="todo">
-      <h2>To Do List</h2>
 
-      <input type="text" id="text-todo" placeholder="Item...." />
-      <button onclick="newItem()">Add</button>
+    <div id="deleteEventModal">
+      <h2>To Do</h2>
 
-      <ul id="list">
-        <li>To Be Determined</li>
-      </ul>
+      <p id="eventText"></p>
+      <button id="completeButton">Complete</button>
+      <button id="deleteButton">Delete</button>
+      <button id="closeButton">Close</button>
```

Comment:
This is a lot of ids. I believe you're using it for Javascript. I think though it'd be better if you used node traversal. (element.children, sibling, parent ...etc)